From 0b0302b5da0a61caa78b875f9f61d6b375e1e0c4 Mon Sep 17 00:00:00 2001
From: Dong Aisheng <b29396@freescale.com>
Date: Thu, 21 May 2015 15:39:52 +0800
Subject: [PATCH] MLK-10944 dts: imx6ul-evk: fix sd1 slot CD pin unwork issue

CD pin is missed to be configured and cd_gpios property is wrong.

Signed-off-by: Dong Aisheng <b29396@freescale.com>
---
 arch/arm/boot/dts/imx6ul-evk.dts | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx6ul-evk.dts b/arch/arm/boot/dts/imx6ul-evk.dts
index ae52ab9..53c3103 100755
--- a/arch/arm/boot/dts/imx6ul-evk.dts
+++ b/arch/arm/boot/dts/imx6ul-evk.dts
@@ -272,7 +272,7 @@
 &usdhc1 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_usdhc1>;
-	cd_gpios = <&gpio1 19 0>;
+	cd-gpios = <&gpio1 19 0>;
 	keep-power-in-suspend;
 	enable-sdio-wakeup;
 	status = "okay";
@@ -472,6 +472,7 @@
 				MX6UL_PAD_SD1_DATA1__USDHC1_DATA1 0x17059
 				MX6UL_PAD_SD1_DATA2__USDHC1_DATA2 0x17059
 				MX6UL_PAD_SD1_DATA3__USDHC1_DATA3 0x17059
+				MX6UL_PAD_UART1_RTS_B__GPIO1_IO19 0x17059 /* CD */
 			>;
 		};
 
-- 
1.8.0

